__tests__/components.test.js

```diff
@@ -124,9 +126,14 @@ describe('Components', () => {
       [/block]`,
       rdmd: `[](https://www.nytimes.com/2020/05/03/us/politics/george-w-bush-coronavirus-unity.html "@embed")`,
     };
-    Object.values(fixtures).map(fx => {
-      const wrap = mount(markdown.react(fx));
-      return expect(wrap.html()).toMatchSnapshot();
+
+    silenceConsole()(error => {
```

Comment:
Is this to quieten the console.logs during testing?

Reply:
Yes, there's a caught error that prints to `error`